Tactical/Overhead.cpp

```diff
@@ -11092,6 +11034,105 @@ void HandleTurncoatAttempt( SOLDIERTYPE* pSoldier )
 	}
 }
 
+void EscapeTimerCallback()
+{
+    const bool chanceToEscape = Chance(75);
+    bool escaped = false;
+    // Look for an escape direction for remaining mercs
+    std::vector<WorldDirections> possibleEscapeDirections{ NORTH, EAST, SOUTH, WEST };
+    std::random_shuffle(possibleEscapeDirections.begin(), possibleEscapeDirections.end());
+
+    for (auto direction : possibleEscapeDirections)
+    {
+        if (IsEscapeDirectionValid(direction) && chanceToEscape && gbWorldSectorZ == 0) // There is no escaping underground! For now..
+        {
+            escaped = true;
+            ScreenMsg(FONT_MCOLOR_LTYELLOW, MSG_INTERFACE, szPrisonerTextStr[STR_PRISONER_ESCAPE]);
+
+            JumpIntoEscapedSector(direction);
+            break;

```

Comment:
if we're here, escaped has been set to true and there's either no need for this break or for the next if statement